modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/serializer/FullComponentDataSerializerProvider.java

```diff
@@ -0,0 +1,14 @@
+package com.enonic.xp.core.impl.content.serializer;
+
+import com.enonic.xp.region.LayoutDescriptorService;
+
+public class FullComponentDataSerializerProvider extends ComponentDataSerializerProvider
+{
+
+    public FullComponentDataSerializerProvider( final LayoutDescriptorService layoutDescriptorService )
+    {
+        super();
+        this.layoutDataSerializer = new FullLayoutComponentDataSerializer( layoutDescriptorService, regionDataSerializer );
```

Comment:
this looks a bit strange that we change filed already assigned. in parent constructor. probably better to have protected constructor with an argument